music21/musicxml/m21ToXml.py

```diff
@@ -6437,7 +6476,10 @@ def testFromScoreNoParts(self):
         s.append(note.Note())
         s.append(note.Note())
         gex = GeneralObjectExporter(s)
-        tree = ET.fromstring(gex.parse().decode('utf-8'))
+
+        warn_msg = f'{s} is not well-formed; see isWellFormedNotation()'
+        with self.assertWarns(MusicXMLWarning, msg=warn_msg):
+            tree = ET.fromstring(gex.parse().decode('utf-8'))
```

Comment:
Looks like I made the same mistake here I cleaned up in ca6f533, so I'll push another commit.